posthog/api/insight.py

```diff
@@ -170,9 +170,14 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                 queryset = queryset.filter(created_by=request.user)
             elif key == "favorited":
                 queryset = queryset.filter(Q(favorited=True))
+            elif key == "date_from":
+                queryset = queryset.filter(updated_at__gt=relative_date_parse(request.GET["date_from"]))
```

Comment:
What happens if these parameters aren't provided? Which defaults would be used?

Reply:
It should refer to no date filter in that case so = all insights